apps/etterlatte-beregning/src/main/kotlin/klienter/BehandlingKlient.kt

```diff
@@ -95,6 +101,38 @@ class BehandlingKlientImpl(
         }
     }
 
+    override suspend fun hentFoersteVirkningsdato(
+        sakId: Long,
+        brukerTokenInfo: BrukerTokenInfo,
+    ): FoersteVirkDto {
+        logger.info("Henter foersteVirkningsdato med saksId=$sakId")
+
+        return retry<FoersteVirkDto> {
```

Comment:
Ikkje noko du treng å gjera no, og det her er jo fint i samsvar med sånn det er gjort elles i denne fila, men generelt synes eg hjelpemetoden `retryOgPakkUt` er fin, den kamuflerer unna litt boilerplate-kode for oss (spesifikt let-blokka under)